packages/access-client/src/types.ts

```diff
@@ -103,7 +107,7 @@ export interface Service {
     info: ServiceMethod<SpaceInfo, SpaceRecord, Failure | SpaceUnknown>
     'recover-validation': ServiceMethod<
       SpaceRecoverValidation,
-      EncodedDelegation<[SpaceRecover]> | undefined,
+      ShouldShowValidationNonce | EncodedDelegation<[SpaceRecover]> | undefined,
```

Comment:
Please can we remove this, and cast it in the tests to appease typescript? I'm not super happy to have types used only for testing in our public facing API.

Reply:
ok chatted out of sync I misunderstood and this is about `EncodedDelegation<[SpaceRecover]>` - I'm going to work on this in a separate PR